### PR TITLE
Content Review: Salt Guide 2/2

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/8365

### CODE REVIEW

**Comment by Loquacity** [comment]
And that completes the Salt Guide! :tada:

**Comment by Loquacity** [comment]
@keichwa @jcayouette can i please get a review on this?

File: modules/salt/pages/salt-ssh.adoc [hunk 1]
```diff
@@ -1,329 +1,305 @@
...
-
-* https://github.com/SUSE/spacewalk/blob/Manager/java/code/src/com/suse/manager/webui/controllers/SaltSSHController.java[com.suse.manager.webui.controllers.SaltSSHController]
-
-* https://github.com/SUSE/spacewalk/blob/Manager/susemanager-utils/susemanager-sls/modules/runners/mgrutil.py[mgrutil.ssh_keygen]
-
-* https://github.com/SUSE/spacewalk/blob/Manager/proxy/proxy/mgr-proxy-ssh-push-init[mgr-proxy-ssh-push-init]
-
-* https://github.com/SUSE/spacewalk/blob/Manager/proxy/proxy/spacewalk-proxy.spec[spacewalk-proxy.spec]
+====
+Salt SSH generates a keypair the first time it is invoked in
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/salt/pages/salt-ssh.adoc]: maybe "with"?

[the remark this answers is not in the document]
Comment (keichwa) [reply modules/salt/pages/salt-ssh.adoc]: or "from within" --but you decide, you are the native speaker :)

File: modules/salt/pages/salt-ssh.adoc [hunk 2]
```diff
@@ -1,329 +1,305 @@
...
-
-
-
-==== Twin Proxies and SSH Push
-
-The following example uses the ProxyCommand option with two proxies and the usual ssh-push method
-
-This is a test.
 
+This example uses `ProxyCommand` with two proxies and the `ssh-push` method:
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/salt/pages/salt-ssh.adoc]: I don't either, but it was the neatest way to do this.

**Pull request merged**